Makefile

```diff
@@ -133,9 +134,48 @@ build-darwin:
 	  $(MAKE) build
 
 build-linux:
-	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/darwin_amd64 DOCKER=1 \
+	SC_GOOS=linux SC_GOARCH=amd64 BINDIR=bin/linux_amd64 DOCKER=1 \
 	  $(MAKE) build
 
+images: build-linux $(shell find pkg contrib -name Dockerfile)
+	# catalog
+	cp pkg/controller/catalog/Dockerfile $(BINDIR)/linux_amd64/
+	docker build -t catalog:$(GIT_VERSION) $(BINDIR)/linux_amd64
+	# registry
+	cp contrib/registry/Dockerfile $(BINDIR)/linux_amd64/
+	cp contrib/registry/data/charts/*.json $(BINDIR)/linux_amd64/
+	docker build -t registry:$(GIT_VERSION) $(BINDIR)/linux_amd64
+	rm -f $(BINDIR)/linux_amd64/*.json
+	# k8s-broker
+	cp contrib/broker/k8s/Dockerfile $(BINDIR)/linux_amd64/
+	docker build -t k8s-broker:$(GIT_VERSION) $(BINDIR)/linux_amd64
+	# user-broker
+	cp contrib/broker/user_provided/Dockerfile $(BINDIR)/linux_amd64/
+	docker build -t user-broker:$(GIT_VERSION) $(BINDIR)/linux_amd64
+	# controller
+	cp pkg/controller/catalog/Dockerfile $(BINDIR)/linux_amd64/
+	docker build -t controller:$(GIT_VERSION) $(BINDIR)/linux_amd64
+	# clean up
+	rm -f $(BINDIR)/linux_amd64/Dockerfile
+
+push: images
```

Comment:
Yes. the old stuff supported all make targets to be used individually (with the exception of `make format`) so one could build and push individual image. For images it was primarily consequence of the overall design and if it is possible to test outside of k8s deployment it is not critical to build individual images. However, if the local experience begins to rely more and more on k8s clusters and deploying things, being able to rebuild an individual image would become much more useful.